docs/advanced/compiler-annotations.md

```diff
@@ -665,3 +509,244 @@ end
 ```
 
 </SideBySide>
+
+## Deprecated
+
+:::warning
+Some annotations are deprecated and will be/have been removed.
+The docs are only valid for older versions and include some instructions on how to replace the annotations with vanilla TS.
+:::
+
+## @extension
+
+<DeprecatedInVersion deprecated="0.37.0" removed="TBD"></DeprecatedInVersion>
```

Comment:
```suggestion
<DeprecatedInVersion deprecated="0.37.0" removed="TBD" />
```

There are multiple lines like this one which should be changed if this change is acceptable